content/user-guide/index.md

```diff
@@ -41,6 +41,46 @@ sudo -E bash -c "$(curl -s https://raw.githubusercontent.com/mysteriumnetwork/no
 In addition to downloading and installing our Node using either of these commands will also
 install additional required dependencies like WireGuard and OpenVPN if you don't have them already.
 
+### Complete installation with Mysterium CLI (if there is no access to the NodeUI)
+
+Mysterium CLI allows you to register node identity, claim your node into my.mysterium.network, set bounty payout address, view NAT traversal status & NAT type, manually start/stop mysterium-node.service (WG) and more.
+
+Unlock and register your identity.
+
+```bash
+identities list # get your node identity address
+identities unlock <your-identity> # unlock your identity before using it
+identities register <your-identity> # register your identity, must be done only once
+```
+
+Claim your node into my.mysterium.network.
+
+```bash
+mmn <your-api-key>
+```
+**Note**: You can find and copy your MMN API key in your [MMN profile](https://my.mysterium.network/user/profile).
+
+Set payout wallet address.
+
+```bash
+identities set-payout-address <providerIdentity> <beneficiary>
+```
+
+### Manage node service with MysteriumCLI (if there is no access to the NodeUI)
+
+```bash
+service start <your-identity> wireguard # start wireguard service
```

Comment:
I think this is started by default, so maybe some better wording here as to not confuse people